src/backend/services/users.js

```diff
@@ -12,7 +18,22 @@ const checkUser = async (email) => {
   return {userRegister: false};
 };
 
+const checkUserPassword = async (email, password) => {
+  const user = await userRepository.getUserPassword(email);
+  return await bcrypt.compare(password, user.dataValues.password);
+};
+
+const getUserInfo = async (email) => await userRepository.getUserInfo(email);
+
+const checkAdmin = async (email) => {
+  const userInfo = await userRepository.getUserInfo(email);
+  return userInfo.dataValues.roleId === 1 ? {isAdmin: true} : {isAdmin: false};
```

Comment:
что за магическое 1 ?
```suggestion
  return {isAdmin: userInfo.dataValues.roleId === ADMIN_ROLE_ID};
```